src/Uno.UI/Uno.UI.Skia.csproj

```diff
@@ -97,6 +97,10 @@
 		</ProjectReference>
 	</ItemGroup>
 
+	<ItemGroup>
+		<PackageReference Include="Silk.NET.OpenGL" Version="2.16.0" />
```

Comment:
This shouldn't be needed here. I see the usages are only in the Uno.UI.Runtime.Skia and Uno.UI.Runtime.Skia.Wpf packages